### PR TITLE
Add English readme

### PR DESCRIPTION
**Pull request by k-shimari**
- CCFinderSW only has a Japanese readme, so I translate the Japanese readme into English.
  - Fix some missing links and modify some phrases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,59 @@
...
-## 利用についての注意点
-CCFSWは字句解析を行いますが，言語ごとに個別の字句解析機構は持っていません．  
-オプションファイルを使用することで各言語に合わせた字句解析を行います．  
-（また同時に，各言語に最適化されたレクサーアルゴリズムではない，ということも意味します．）  
-このオプションファイルはユーザ自身が変更できるものです．  
-オプションファイルの記法については, [OptionFile.md](UsageJp/OptionFile.md)  を参照してください． 
+This software contains libraries distributed under Apache Licence 2.0.
+
+## Other Information
+Author: Yuichi Semura
```
Comment (YuichiSemura) [inline README.md]: add 2 whitespace in end of this line

File: Usage/Run.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+   - 2 Between files only
+- -b
+   - Recognizes code blocks with blank indentation.
+   - Cannot be used in antlr mode.
+- -antlr \<arg>
+   - Extracts information used for comment removal and identifier identification from the grammar files in `src/main/dist/grammarsv4`.
+   - Normal rules cannot be used.
+   - The argument must be a regular expression of the extension name to recognize the target source code.  (just put the extensions in a row with a vertical bar between).
+       - Example: -antlr h|hh|hpp|hxx|c|cc|cpp|cxx  
+                  -antlr py
```
Comment (YuichiSemura) [inline Usage/Run.md]: please rewrite l42-l43
```
       - Example1: -antlr h|hh|hpp|hxx|c|cc|cpp|cxx  
       - Example2: -antlr py
```

**Review by YuichiSemura — COMMENTED** [review]
Thanks shimari sensei!!
I reviewed only looks, sorry.
Please solve these problems.

In the original documents, many problems remain... :-<

File: README.md (hunk 1 above)
Comment (k-shimari) [reply README.md]: Thanks for your review.
I have added two whitespaces at the end of the line in the Japanese and English Readme files.

File: Usage/Run.md (hunk 2 above)
Comment (k-shimari) [reply Usage/Run.md]: Thanks for your review.
I have updated this part in the Japanese and English Readme files.

**Pull request merged**